build.sh

```diff
@@ -663,27 +671,33 @@ function processSource {
     cargo run "${cargo_args[@]}" <"$HTML_SOURCE/$source_location" >"$HTML_TEMP/source-whatwg-complete"
   fi
 
-  runWattsi "$HTML_TEMP/source-whatwg-complete" "$HTML_TEMP/wattsi-output"
-  if [[ $WATTSI_RESULT == "0" ]]; then
-    if [[ $LOCAL_WATTSI != "true" ]]; then
-      "$QUIET" || grep -v '^$' "$HTML_TEMP/wattsi-output.txt" # trim blank lines
-    fi
+  if [[ $USE_BIKESHED == "true" ]]; then
+    echo "BIKESHED!!!"
+    bikeshed spec --byos "$HTML_TEMP/source-whatwg-complete" "$HTML_TEMP/bikeshed-output" --md-Text-Macro="SHA $HTML_SHA"
```

Comment:
Not sure if `--byos` (bring your own spec) helps here, didn't try with and without.